rosetta-source/src/main/resources/cdm-sample-files/functions/repo-and-bond/repo-substitution.md

```diff
@@ -0,0 +1,10 @@
+### Create Substitution Business Event
+
+#### New Execution
+- Execution is performed on `2021-3-18` between parties `Global Bank Inc (Seller)` and `UK Bank plc (Buyer)`
+- `10000000` notional of `GB00B24FF097` are sold with dirty price of `1.0075 GBP`, a repo rate of `0.4%` and haircut of `1%`
+- Purchase date is on `2021-3-19` and repurchase date `2021-3-22`
+
+#### Substitution
+- A substitution of collateral occurs that replaces `GB00B24FF097` with `'GB00BMGR2916' at a dirty price of 98.5 an amount of 10,228,426.
```

Comment:
New security code should be surrounded by back ticks and not apostrophes in order to match the rest of the text